docs/source-fabric/fundamentals/installation.rst

```diff
@@ -37,7 +37,8 @@ Fabric is part of the `Lightning <https://lightning.ai>`_ package. Here is how y
 
 
 If you don't already have it, this command will also install the latest `stable PyTorch version <https://pytorch.org/>`_.
-The oldest version of PyTorch that Fabric supports is v1.11.0.
+
+You can find our the list of supported PyTorch versions in our `compatibility matrix <https://lightning.ai/docs/pytorch/latest/versioning.html#compatibility-matrix>`__.
```

Comment:
I don't think fabric docs can link to PL docs, so I used a normal hyperlink

Reply:
That's perfectly fine!